pycds/__init__.py

```diff
@@ -26,18 +26,57 @@
 fail with errors of the form "could not find object X in schema Y".
 """
 
-import os
-import datetime
-
+# TODO: Do we really need __all__? Its only use is for `from <package> import *`
```

Comment:
You're right that we don't *need* to, and yes, `import *` is generally discouraged.

However, "discouraged" isn't the same as "should *never* be done" and there is some merit to explicitly defining what a modules public exports are supposed to be. Without doing so, there's a much higher likelihood of namespace collisions. Especially, when this module imports widely used things like `Base`, `String` or `os`. The narrower the namespace of the public interface, the better.